nbdime/merge_format.schema.json

```diff
@@ -0,0 +1,144 @@
+
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Merge decisions",
+  "description": "A list of merge decisions that in their entirety describes a merge of two Jupyter Notebooks",
+  "type": "array",
+  "items": {"$ref": "#/definitions/decision"},
+
+  "definitions": {
+    "decision": {
+      "type": "object",
+      "description": "An individual merge decision",
+      "additionalProperties": false,
+      "properties": {
+        "local_diff": {
+          "description": "Local diff description relevant for this decision",
+          "$ref": "#/definitions/diff"
+        },
+
+        "remote_diff": {
+          "description": "Remote diff description relevant for this decision",
+          "$ref": "#/definitions/diff"
+        },
+
+        "conflict": {
+          "type": "boolean",
+          "description": "Whether the entry is conflicted or not"
+        },
+
+        "action": {
+          "description": "How the merge was resolved, or for conflicted entries, the \"best\" initial suggestion for resolution",
+          "enum": ["local", "remote", "base", "clear", "custom"]
```

Comment:
What about "local_then_remote" and "remote_then_local"? Or would you cover that by combining the diffs into "custom_diff"?

And "either"? Although that's recoverable by checking if local_diff == remote_diff.